lib/git-shell-out-strategy.js

```diff
@@ -380,7 +380,7 @@ export default class GitShellOutStrategy {
   }
 
   commit(message, {allowEmpty, amend} = {}) {
-    let args = ['commit'];
+    let args = ['commit', '--cleanup=strip'];
```

Comment:
Do we definitely want this all the time? For command-line Git, `-m` messages have `cleanup` set to `whitespace`. I guess the option would be to pass `--cleanup=strip` only when using the larger commit editor.... but it also seems reasonable that we'd wanna strip comments from the mini-editor too. (Do we syntax highlight the mini editor?)

Reply:
Yeah I think so. As of now, in the commit box we're already stripping out lines starting with `#`. In this way we deviate from how committing with `-m` works. The nice thing about doing this all the time is that we will have consistent behavior between the commit box and the expanded editor. We don't syntax highlight the mini editor.